AdyenUI/UI/Form/Items/Value/FormValueItemView.swift

```diff
@@ -17,7 +17,9 @@ open class FormValueItemView<ValueType, Style, ItemType: FormValueItem<ValueType
 
     /// The top label view.
     public lazy var titleLabel: UILabel = {
-        let titleLabel = UILabel(style: item.style.title)
+        let titleLabel = UILabel()
+        titleLabel.font = AdyenTheme().currentFonts.bodyEmphasized
+        titleLabel.textColor = defaultTitleColor
```

Comment:
What's the semantic meaning for `default` here? We don't have any other color in this local context, can it be just `titleColor`? 
Any difference between `.font` and `.textColor`, why does one deserve local var and other doesn't?